entity-framework/core/what-is-new/ef-core-5.0/whatsnew.md

```diff
@@ -0,0 +1,125 @@
+---
+title: What's New in EF Core 5.0
+author: ajcvickers
+ms.date: 01/29/2020
+uid: core/what-is-new/ef-core-5.0/whatsnew.md
+---
+
+# What's New in EF Core 5.0
+
+EF Core 5.0 is currently in development.
+This page will contain an overview of interesting changes introduced in each preview.
+The first preview of EF Core 5.0 is tentatively expected in the next couple of months.
```

Comment:
Does the page end up actually being date-stamped such that "next couple of months" means anything?

Reply:
I'll change the wording. I don't want it to be too specific because we might change plans around previews. We sometimes do that. ;-)